server/src/projects/projects.service.ts

```diff
@@ -15,8 +15,15 @@ export class ProjectsService {
   public async findManyByContactId(contactId: string) {
     let results = null;
 
+    const applicantActiveStatusCode = '1';
+    const projectActiveStateCode = '0';
+    const packageVisibilityApplicantOnly = '717170002';
+    const packageVisibilityGeneralPublic = '717170003';
+    const projectVisibilityApplicantOnly = '717170002';
+    const projectVisibilityGeneralPublic = '717170003';
+
     try  {
-      results = await this.crmService.get(`dcp_projects?$filter=%20dcp_dcp_project_dcp_projectapplicant_Project/%20any(o:o/_dcp_applicant_customer_value%20eq%20%20%27${contactId}%27)%20&$expand=%20dcp_dcp_project_dcp_package_project,%20dcp_dcp_project_dcp_projectapplicant_Project`);
+      results = await this.crmService.get(`dcp_projects?$filter=dcp_dcp_project_dcp_projectapplicant_Project/any(o:o/_dcp_applicant_customer_value%20eq%20%27${contactId}%27)%20and%20(dcp_visibility%20eq%20${projectVisibilityApplicantOnly}%20or%20dcp_visibility%20eq%20${projectVisibilityGeneralPublic})&$expand=dcp_dcp_project_dcp_package_project($filter=%20statecode%20eq%20${projectActiveStateCode}%20and%20(dcp_visibility%20eq%20${packageVisibilityApplicantOnly}%20or%20dcp_visibility%20eq%20${packageVisibilityGeneralPublic})),dcp_dcp_project_dcp_projectapplicant_Project($filter=%20statecode%20eq%20${applicantActiveStatusCode})`);
```

Comment:
What are some ways to make this easier to read?

Reply:
Good point! Will look into it and address in separate PR :)